src/documentation/explore/ToolkitBreadcrumbHeading.tsx

```diff
@@ -63,17 +63,8 @@ const ToolkitBreadcrumbHeading = ({
           </Text>
         </Text>
       </Button>
-      <HStack align="flex-start" spacing={4}>
-        {icon && (
-          <Image
-            src={imageUrlBuilder.image(icon.asset).url()}
-            alt=""
-            width="80px"
-            height="64px"
-            borderRadius="lg"
-            mt={1}
-          />
-        )}
+      <HStack align="center" spacing={4}>
```

Comment:
I'm a bit unsure on `align="center"` here, but it seems to the yield the best results with inbuilt spacing inside the icons.

Reply:
Created a layout issue on smaller screen/sidebar widths are has been resolved with [`e6c13b6`](https://github.com/microbit-foundation/python-editor-next/pull/504/commits/e6c13b656f5a8aa0e08c7c1218a12e678f23a98f).